### PR TITLE
Update paywall3.md

### PR DESCRIPTION
**Pull request by liljanadimovska**
Added info about paywall standalone donation class

### CODE REVIEW

File: docs/paywall3.md [hunk 1]
```diff
@@ -699,6 +699,7 @@ Here is the list of the HTML classes that can be used for creating each of the s
 | `inplayer-paywall-ccm`      | Invokes the  default card management screen |
 | `inplayer-paywall-change-pass`  | Invokes the 'Change Password' screen |
 | `inplayer-paywall-purchases`  | Invokes the 'My Purchases' paywall screen where an end-user can see the list of payments they have completed |
+| `inplayer-paywall-donations`	| Invokes the donations flow for the provided asset  |
```
Comment (matejpetrov) [inline docs/paywall3.md:702]: @liljanadimovska , let's discuss tomorrow if we should:

1. add another column in this table where we would document the ID
OR
2. leave a * at the end of the class and provide a more elaborate description of the usage below the table

File: docs/paywall3.md [hunk 2]
```diff
@@ -750,6 +751,17 @@ Here are listed and defined the Paywall 3.0 standalone functions:
 
 Table 4
 
+### `donations`
+
+This HTML class can be added to any HTML element. The `asset-id` represents the ID of a specific InPlayer asset. Usage example:
```
Comment (matejpetrov) [inline docs/paywall3.md:756]: This is a great and elaborate description, we should however provide it under the table in the _**Standalone Functionalities**_ section. 

Let's move this section there

File: docs/paywall3.md [hunk 3]
```diff
@@ -750,6 +751,17 @@ Here are listed and defined the Paywall 3.0 standalone functions:
...
+
+```
+<button class="inplayer-paywall-donations" data-asset-id="{asset-id}" />
+```
+
+When clicking on this element, the donations flow for the provided asset is initiated by opening the donation options screen. 
+The donation options screen lists all donation options added to the asset that is being accessed. Upon selecting a donation option, with the user being logged in, the payment screen with the card payment method only available for donation will be shown. Once the payment is complete, a screen with a thank you message is displayed.
```
Comment (matejpetrov) [inline docs/paywall3.md:763]: I think the description is good, but too long. 

My suggestion:
... asset that is being accessed. After selecting a donation option the end-user can proceed with the donation payment.

What do you think?

File: docs/paywall3.md (hunk 1 above)
Comment (liljanadimovska) [reply docs/paywall3.md]: @matejpetrov updated

**Review by matejpetrov — APPROVED** [review]
Changes look great

**Pull request merged**